mapper/pojo-base/src/main/java/org/hibernate/search/mapper/pojo/mapping/definition/annotation/processing/MappingAnnotationProcessorContext.java

```diff
@@ -89,4 +92,8 @@
 	 */
 	EventContext eventContext();
 
+	static Map<String, Object> toMap(Param[] params) {
```

Comment:
All other helpers (`toPojoModelPathValueNode`, `toBeanReference`, ...) are instance methods... why would this one be static?

Also... some javadoc, please? :)